remix/app/routes/newsletters.tsx

```diff
@@ -0,0 +1,61 @@
+import type { User } from "@prisma/client";
+import type { LoaderArgs } from "@remix-run/node";
+import {
+  Form,
+  isRouteErrorResponse,
+  useLoaderData,
+  useRouteError,
+} from "@remix-run/react";
+import { SocialsProvider } from "remix-auth-socials";
```

Comment:
Let's ada line break between third-party (node module) imports and application imports

```suggestion
import { SocialsProvider } from "remix-auth-socials";

```